deploy/helm/sumologic/values.yaml

```diff
@@ -99,6 +99,10 @@ sumologic:
   metrics:
     enabled: true
 
+  ## Configuration of additional collector fields
+  ## https://help.sumologic.com/Manage/Fields#http-source-fields
+  fields: {}
```

Comment:
Since this is part of setup only, should it leave in the setup section?

Reply:
+1

Reply:
+1

Reply:
I coukd, but now `setup` looks like well-structured config, and adding `fields` will break it. I am going to add commit which we can revert eventually